libs/cases/frontend/shell/src/lib/shell.routes.ts

```diff
@@ -37,7 +37,7 @@ export const ORDERS_SHELL_ROUTES: Route[] = [
 					},
 				},
 				loadComponent: () =>
-					import('orders-frontend-view').then((m) => m.OpenOrdersComponent),
+					import('cases-frontend-view').then((m) => m.OpenCasesComponent),
 			},
 			{
 				// Route to detail order
```

Comment:
In the following lines and in other positions you still have to change the url: to 'cases/...'
right now the breadcrumbs are leading to the not found page

Reply:
... in this whole file

Reply:
I also noticed and forgot to mention something that was wrong before so the url here actually has to be 'cases/create' and not just cases
breadcrumbs: {
label: 'Create',
url: 'create',
},